support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/ContributionThankYou.jsx

```diff
@@ -174,9 +174,11 @@ const ContributionThankYou = ({
     />,
     shouldShow: contributionType === 'ONE_OFF',
   };
+  const SURVEY_END_DATE = new Date(2020, 10, 5);
```

Comment:
can you use `new Date(Date.parse('2020-11-05'))` please
The off-by-one nature of months is too confusing!

Reply:
great shout! it's so confusing haha 😂